modules/farm/farm_access/farm_access.install

```diff
@@ -43,3 +43,8 @@ function farm_access_update_7000(&$sandbox) {
     module_enable(array($module));
   }
 }
+
+/**
+ * Clean up variables.
+ */
+variable_del('farm_access_allow_origin');
```

Comment:
This needs to be inside `hook_uninstall()` so it only runs when the module is uninstalled.